Files/Views/Pages/PropertiesSecurityAdvanced.xaml

```diff
@@ -155,7 +155,7 @@
                 x:Name="ListView"
                 Grid.Row="2"
                 x:Load="{x:Bind ViewModel.FilePermissions.CanReadFilePermissions, Mode=OneWay}"
-                Background="{ThemeResource SolidBackgroundAcrylic}"
+                Background="{ThemeResource ControlFillColorDefaultBrush}"
```

Comment:
Maybe it was there before but is it possible to fix the background color of the list header? Or remove the margins.

![image](https://user-images.githubusercontent.com/9673091/127751047-572b858f-8024-429e-bd8b-55c6d6d1c50b.png)

Reply:
I updated it to be tinted with the control color instead of a hard coded color, and also remove the margin.

![image](https://user-images.githubusercontent.com/59544401/127752048-38f4caf4-7e79-44d5-b337-eec733effd2f.png)